src/components/Header/LogoAndMode/Logo.tsx

```diff
@@ -0,0 +1,20 @@
+import styles from '@/components/Header/LogoAndMode/Logo.module.css';
+import logo from '@/img/logo.svg';
+import divider from '@/img/mode/divider.svg';
+import moon from '@/img/mode/moon.svg';
+import sun from '@/img/mode/sun.svg';
+import sunBlack from '@/img/mode/sun-black.svg';
+
+const Logo = () => (
```

Comment:
The name of your component should be the same as a directory where it located. And css module should have the same name too. 

![image](https://github.com/DavidTereshchenko/fe-react-2024/assets/108006836/308f6c16-5bf1-48d6-82e4-53424588a04d)